src/flycapture2.pyx

```diff
@@ -385,6 +394,165 @@ cdef class Context:
         with nogil:
             r = fc2SetConfiguration(self.ctx, &config)
         raise_error(r)
+        
+        
+    ############################################################
+    # GigE support
+
+    def set_byteorder_littleendian(self):
+        '''Get byte order        
+        '''
+        cdef fc2Error r
+        cdef unsigned int regval = 0        
+        
+        with nogil:    
+            r = fc2WriteRegister(self.ctx, 0x1048, regval)
```

Comment:
Yes. Please remove those two methods and expose the functionality as `write_register(address, value)` and `read_register(address)`. That's more generic as it allows setting other registers as well.
And it's not true that data is "scrambled" when big endian. It's just that: big endian. That's completely fine with e.g. numpy which can happily interpret big endian arrays. And it is also the natural endianess on many architectures.